_posts/2017-04-04-things-the-rust-community-team-does.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Things the Rust Community Team does"
+author: Florian Gilcher
+description: "Describing the Rust Community Team work"
+---
+
+A common question that we get is "what precisely does the Rust community team do?" and probably the member you are currently speaking to has no straight answer to you. That has a reason: the team does a lot of things and any individual often just does a couple of them. Many of the projects that come out of this are then taken on by individual teams and are not necessarily visible as community team action. An important part is to discuss our and others ideas and support each other in our endeavours.
+
+Still, where there's no information, misconceptions arise, in all directions: either that the team doesn't do anything at all or that the teams size and work is way exaggerated.
+
+So, I recently accidentally wrote a pretty exhausive list, which I'd like to extend and republish here. A lot of what we do is all over the place and in a DIY fashion, so we learn a lot of skills on the go. If you have experience with any of these - please get involved!
+
+## Tend a calendar
+
+Perhaps the oldest piece of community documentation is the [Rust community calendar](https://calendar.google.com/calendar/embed?src=apd9vmbc22egenmtu5l6c5jbfc@group.calendar.google.com&pli=1), created by Erick and maintained to this date. Ping us to get access. We regularly check if recurring events are still on.
+
+## Watch the Forums
+
+We read and engage in the comment sections. First of all, we believe that the project should be engaged into discussions about it. Hat tip to Sri Ramkrishna for giving a whole talk about that subject: [Swimming with chum in shark infested water](https://www.youtube.com/watch?v=VYrL_l984-E). This engagement must be complex and assertive.
```

Comment:
I think you need a little more to justify the "This engagement must be complex and assertive". Part of this feels like you are getting into mod-team territory, but I'm not sure if that's what you intend.